content/source/docs/cloud/run/cli.html.md

```diff
@@ -86,6 +86,12 @@ If you manage your Terraform configurations in self-contained repositories, the
 
 If you use a combined repository and [specify a working directory on workspaces](../workspaces/settings.html#terraform-working-directory), you can run Terraform from either the real working directory or from the root of the combined configuration directory. In both cases, Terraform will upload the entire combined configuration directory.
 
+## Excluding files from upload
```

Comment:
```suggestion
## Excluding Files from Upload
```

Title-case for headers.

Reply:
(Sidenote: while I was second-guessing myself on that "from," I found this, which IMHO is excellent. https://titlecaseconverter.com/words-to-capitalize/?style=AP,CMOS,MLA,NYT,WP#from)